src/repository/exercises-repository.ts

```diff
@@ -1304,1527 +1282,3122 @@ export const db: Schema = {
+        Tu: 'permaneces',
+        'Ela/Ele/Você': 'permanece',
+        Nós: 'permanecemos',
+        'Eles/Elas/Vocēs': 'permanecem'
+      },
+      pastPerfect: {
+        Eu: 'permaneci',
+        Tu: 'permaneceste',
+        'Ela/Ele/Você': 'permaneceu',
+        Nós: 'permanecemos',
+        'Eles/Elas/Vocēs': 'permaneceram'
+      }
+    },
+    {
+      english: 'to need',
+      infinitive: 'precisar',
+      presentSimple: {
+        Eu: 'preciso',
+        Tu: 'precisas',
+        'Ela/Ele/Você': 'precisa',
+        Nós: 'precisamos',
+        'Eles/Elas/Vocēs': 'precisam'
+      },
+      pastPerfect: {
+        Eu: 'precisei',
+        Tu: 'precisaste',
+        'Ela/Ele/Você': 'precisou',
+        Nós: 'precisámos',
+        'Eles/Elas/Vocēs': 'precisaram'
+      }
+    },
+    {
+      english: 'to hear',
+      infinitive: 'ouvir',
+      presentSimple: {
+        Eu: 'ouço/oiço',
+        Tu: 'ouves',
+        'Ela/Ele/Você': 'ouve',
+        Nós: 'ouvimos',
+        'Eles/Elas/Vocēs': 'ouvem'
+      },
+      pastPerfect: {
+        Eu: 'ouvi',
+        Tu: 'ouviste',
+        'Ela/Ele/Você': 'ouviu',
+        Nós: 'ouvimos',
+        'Eles/Elas/Vocēs': 'ouviram'
+      }
+    },
+    {
+      english: 'to ask',
+      infinitive: 'pedir',
+      presentSimple: {
+        Eu: 'peço',
+        Tu: 'pedes',
+        'Ela/Ele/Você': 'pede',
+        Nós: 'pedimos',
+        'Eles/Elas/Vocēs': 'pedem'
+      },
+      pastPerfect: {
+        Eu: 'pedi',
+        Tu: 'pediste',
+        'Ela/Ele/Você': 'pediu',
+        Nós: 'pedimos',
+        'Eles/Elas/Vocēs': 'pediram'
+      }
+    },
+    {
+      english: 'to offer',
+      infinitive: 'oferecer',
+      presentSimple: {
+        Eu: 'ofereço',
+        Tu: 'ofereces',
+        'Ela/Ele/Você': 'oferece',
+        Nós: 'oferecemos',
+        'Eles/Elas/Vocēs': 'oferecem'
+      },
+      pastPerfect: {
+        Eu: 'ofereci',
+        Tu: 'ofereceste',
+        'Ela/Ele/Você': 'ofereceu',
+        Nós: 'oferecemos',
+        'Eles/Elas/Vocēs': 'ofereceram'
+      }
+    },
+    {
+      english: 'to obtain',
+      infinitive: 'obter',
+      presentSimple: {
+        Eu: 'obtenho',
+        Tu: 'obténs',
+        'Ela/Ele/Você': 'obtém',
+        Nós: 'obtemos',
+        'Eles/Elas/Vocēs': 'obtêm'
+      },
+      pastPerfect: {
+        Eu: 'obtive',
+        Tu: 'obtiveste',
+        'Ela/Ele/Você': 'obteve',
+        Nós: 'obtivemos',
+        'Eles/Elas/Vocēs': 'obtiveram'
+      }
+    },
+    {
+      english: 'to occur',
+      infinitive: 'ocorrer',
+      presentSimple: {
+        Eu: 'ocorro',
+        Tu: 'ocorres',
+        'Ela/Ele/Você': 'ocorre',
+        Nós: 'ocorremos',
+        'Eles/Elas/Vocēs': 'ocorrem'
+      },
+      pastPerfect: {
+        Eu: 'ocorri',
+        Tu: 'ocorreste',
+        'Ela/Ele/Você': 'ocorreu',
+        Nós: 'ocorremos',
+        'Eles/Elas/Vocēs': 'ocorreram'
+      }
+    },
+    {
+      english: 'to move',
```

Comment:
is more common to use "mudar"
I would say to write "to move / to change" and delete this one